msi/GitChooseEnvironmentDlg.wxs

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I agree about the "classes" of users. And lately, I've been testing the UI. But, yes, it's in my list to test silent installs as well as admin installs. Hmm I just had an awesome idea that may allow for the best of both worlds in terms of "feature" selection--from here on out, post-install configuration. I'll post over on the main issue.